lndr-backend/src/Lndr/Handler/Credit.hs

```diff
@@ -45,18 +46,19 @@ import           Text.Printf
 
 
 lendHandler :: CreditRecord -> LndrHandler NoContent
-lendHandler creditRecord = submitHandler $ creditRecord { submitter = creditor creditRecord }
+lendHandler creditRecord = submitHandler 0 $ creditRecord { submitter = creditor creditRecord }
 
 
 borrowHandler :: CreditRecord -> LndrHandler NoContent
-borrowHandler creditRecord = submitHandler $ creditRecord { submitter = debtor creditRecord }
+borrowHandler creditRecord = submitHandler 0 $ creditRecord { submitter = debtor creditRecord }
 
 
-submitHandler :: CreditRecord -> LndrHandler NoContent
-submitHandler signedRecord@(CreditRecord creditor debtor _ memo submitterAddress _ hash sig _ _ _ _) = do
+submitHandler :: Integer -> CreditRecord -> LndrHandler NoContent
+submitHandler nonceIncrement signedRecord@(CreditRecord creditor debtor _ memo submitterAddress _ hash sig _ _ _ _) = do
```

Comment:
I don't think adding a `nonceIncrement` parameter should be necessary. Why did you do this?